apps/fp-frontend-app/src/fagsak/components/SidePanelKnapp.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { SidebarRightIcon } from '@navikt/aksel-icons';
+import { Button } from '@navikt/ds-react';
+
+import styles from './fagsakGrid.module.css';
+
+interface Props {
+  toggleSideMeny: () => void;
+  visSideMeny: boolean;
+  style?: React.CSSProperties;
+}
+
+export const SidePanelKnapp = ({ toggleSideMeny, visSideMeny, style }: Props) => {
```

Comment:
Denne komponenten trur eg du får laga to versjonar av. Sånn som du har gjort det no så importerar du komponenten (som ligg i app) frå ei pakke. App skal kunne importera pakker, men ikkje omvendt. Eg har sjekka inn ein fiks så du vil no få ein feil på import om du pullar denne branchen.

Det kan vera greit å ha to versjonar av denne sidan dei er iallfall litt ulike, men forskjellig tekst og style. Evt må ein legga den i ei pakke som både app og pakka som brukar denne kan importera

Reply:
Trur det er ein enkel fiks på dette:
  const isWrappedUnder = useMediaQuery({ maxWidth: 1408 });
 <div className={`${styles.rightColumn} ${visSideMeny || isWrappedUnder ? styles.show : styles.hide}`}>

Altso, når skjermbredden blir så smal at ein wrappar under => vis sidemeny

Reply:
Jeg har laget to versjoner av knappen siden de er ulike. Greit å ha det oversiktlig. :)